proxysql/dockerdir/usr/bin/add_pxc_nodes.sh

```diff
@@ -66,7 +70,6 @@ function main() {
         --remove-all-servers \
         --disable-updates \
         --force \
-        --debug \
```

Comment:
@hors are we OK to remove `--debug`? 

@zlcnju does `proxysql-admin` print passwords with `--debug`?

Reply:
I think it is stable enough to remove --debug in default， if we want to debug, we can execute the command in the container and proxysql-admin print passwords with --debug.